### ISSUE TITLE
The operator 'aten::linspace.out' is not currently supported on the ocl backend.

### ISSUE THREAD

**Issue by karolherbst** [artyom-beilis/pytorch_dlprim#41]
hit with some random torch demo I picked from the internet.

It also hits `The operator 'aten::_copy_from_and_resize' is not currently supported on the ocl backend.`

```python
#!/bin/env python3
# -*- coding: utf-8 -*-

import torch
import math


torch.ops.load_library("build/libpt_ocl.so")
torch.utils.rename_privateuse1_backend('ocl')

dtype = torch.float
device = torch.device("ocl")
# device = torch.device("cuda:0") # Uncomment this to run on GPU

# Create random input and output data
x = torch.linspace(-math.pi, math.pi, 2000, device=device, dtype=dtype)
y = torch.sin(x)

# Randomly initialize weights
a = torch.randn((), device=device, dtype=dtype)
b = torch.randn((), device=device, dtype=dtype)
c = torch.randn((), device=device, dtype=dtype)
d = torch.randn((), device=device, dtype=dtype)

learning_rate = 1e-6
for t in range(2000):
    # Forward pass: compute predicted y
    y_pred = a + b * x + c * x ** 2 + d * x ** 3

    # Compute and print loss
    loss = (y_pred - y).pow(2).sum().item()
    if t % 100 == 99:
        print(t, loss)

    # Backprop to compute gradients of a, b, c, d with respect to loss
    grad_y_pred = 2.0 * (y_pred - y)
    grad_a = grad_y_pred.sum()
    grad_b = (grad_y_pred * x).sum()
    grad_c = (grad_y_pred * x ** 2).sum()
    grad_d = (grad_y_pred * x ** 3).sum()

    # Update weights using gradient descent
    a -= learning_rate * grad_a
    b -= learning_rate * grad_b
    c -= learning_rate * grad_c
    d -= learning_rate * grad_d


print(f'Result: y = {a.item()} + {b.item()} x + {c.item()} x^2 + {d.item()} x^3')
```

**Comment by artyom-beilis** [comment]
Ok... I think I fixed it. Hopefully did correctly. I'll check if further

Fixed in be908fd875410df72909b14727ca23a8371b42cf

**Comment by artyom-beilis** [comment]
Just small note it still warns on some missing ops and falls on CPU but it does not fails due to `_copy_from_and_resize`
